projects/real_world_ovmm/evalai_env.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from datetime import datetime
+from typing import Optional, Tuple
+
+import click
+import evaluation_pb2
+import evaluation_pb2_grpc
+import grpc
+import rospy
+
+from home_robot.agent.ovmm_agent.ovmm_agent import OpenVocabManipAgent
+from home_robot.motion.stretch import STRETCH_HOME_Q
+from home_robot_hw.env.stretch_pick_and_place_env import StretchPickandPlaceEnv
+from home_robot_hw.utils.config import load_config
+
+
+def grpc_dumps(entity):
+    return pickle.dumps(entity)
+
+
+def grpc_loads(entity):
+    return pickle.loads(entity)
+
+
+class Environment(evaluation_pb2_grpc.EnvironmentServicer):
```

Comment:
Should this go in real_world_ovmm, or should we put it in home_robot (if it does not require ROS) or in home_robot_hw?

Reply:
Say src/home_robot_hw/home_robot_hw/utils/eval.py